ui/lib/types.ts

```diff
@@ -37,12 +37,15 @@ export enum V2Routes {
   ImageAutomationUpdatesDetails = "/image_update",
   ImageAutomationRepositoryDetails = "/image_repo",
   ImagePolicyDetails = "/image_policy",
-
+  Policies ="/policies",
+  PolicyDetailsPage="/policy_details",
   //Image updates subRoute
   ImageRepositories = "/image_automation/repositories",
   ImagePolicies = "/image_automation/policies",
   ImageUpdates = "/image_automation/updates",
 
+  PolicyDetails="/policies/details",
```

Comment:
Do we use both  `PolicyDetailsPage` & `PolicyDetails` ?

Reply:
no need for `PolicyDetails`  I forgot to remove it.